packages/core/EventManager.ts

```diff
@@ -574,24 +569,16 @@ export default class EventManager {
             (c) => c.type === destination.integration
           );
           // It might not be the first connected calendar as it seems that the order is not guaranteed to be ascending of credentialId.
-          const firstCalendarCredential = destinationCalendarCredentials[0];
+          const firstCalendarCredential = destinationCalendarCredentials[0] as
```

Comment:
It could be undefined as well. So, explicitly assert it as TypeScript by default can't do that.